snuba/datasets/configuration/spans/entities/metrics_summaries.yaml

```diff
@@ -0,0 +1,84 @@
+version: v1
+kind: entity
+name: metrics_summaries
+
+schema:
+  [
+    { name: project_id, type: UInt, args: { size: 64 } },
+    { name: span_id, type: UInt, args: { size: 64 } },
+    { name: trace_id, type: UUID },
+    { name: metric_name, type: String },
+    { name: min, type: Float, args: { size: 64 } },
+    { name: max, type: Float, args: { size: 64 } },
+    { name: sum, type: Float, args: { size: 64 } },
+    { name: count, type: Float, args: { size: 64 } },
+    { name: end_timestamp, type: DateTime },
+    {
+      name: tags,
+      type: Nested,
+      args:
+        {
+          subcolumns:
+            [{ name: key, type: String }, { name: value, type: String }],
+        },
+    },
+    {
+      name: _tags_hash_map,
+      type: Array,
+      args:
+        {
+          inner_type: { type: UInt, args: { size: 64 } },
+          schema_modifiers: [readonly],
+        },
+    },
+    { name: partition, type: UInt, args: { size: 16 } },
+    { name: offset, type: UInt, args: { size: 64 } },
+    { name: retention_days, type: UInt, args: { size: 16 } },
+    { name: deleted, type: UInt, args: { size: 8 } },
+  ]
+
+storages:
+  - storage: spans
+    is_writable: true
+    translation_mappers:
+      subscriptables:
+        - mapper: SubscriptableMapper
+          args:
+            from_column_table: null
+            from_column_name: tags
+            to_nested_col_table: null
+            to_nested_col_name: tags
+            value_subcolumn_name: value
+            nullable: false
+
+storage_selector:
+  selector: DefaultQueryStorageSelector
+
+query_processors:
+  - processor: TimeSeriesProcessor
```

Comment:
I would love for this processor to die. It is mainly syntactic sugar around the `GRANULARITY` clause of SnQL. If you specify a `GRANULARITY 3600` and reference the magic column `time`, it converts it to `toStartOfHour(timestamp)` or whatever. It causes confusion with people all the time and really provides little value.

If you really want it, then you can leave this in, but if you don't think you'll need it I would ask it be removed. Everyone keeps copy and pasting it from other entities.

Reply:
I removed it, this was definitely a copy/paste from another configuration.

Reply:
Thanks!